apps/src/componentLibrary/tabs/_Tab.tsx

```diff
@@ -23,10 +25,16 @@ export interface TabModel {
   tooltip?: TooltipProps;
   /** Tab icon */
   icon?: FontAwesomeV6IconProps;
+  /** Tab size (e.g. Used for closableButton) */
+  size?: ComponentSizeXSToL;
```

Comment:
should this be `closeButtonSize` since it's only used for the close button and not the tab itself?

Reply:
I think it's better to leave it `size` since it actually indicates the size of current tabs (tab size is set in `Tabs.tsx`). Though now it only used to set size of `CloseButton` in `_Tab.tsx` , in the future it still might be use for other size related aspects